hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml

```diff
@@ -5117,6 +5117,14 @@
     </description>
   </property>
 
+  <property>
+    <name>yarn.router.asc-interceptor-max-size</name>
+    <value>1048576</value>
+    <description>
+
```

Comment:
Can we add a description and fix the blank warning?

Reply:
Thank you very much for your help to review the code, I will modify the code.